frontend/types/index.ts

```diff
@@ -32,6 +32,69 @@ export enum ERole {
   "USER" = "USER",
 }
 
+interface IBuildLinks {
+  cover: {
+    href: string
+    width: number
+    height: number
+  }
+  self: {
+    href: string
+  }
+  versions: {
+    href: string
+  }
+  "add-version": {
```

Comment:
would you prefer snake_case for the rels to avoid quotes?

Reply:
I think you mean camel?

Reply:
I did mean snake, as the rest of the json is also using snake case. I think snake is in general slightly more readable than camel, but to be honest I don't have a strong opinion. It's usually just my default choice. Would you prefer switching everything to camel? Note that the factorio api also uses snake case.

Reply:
ahhh then yes, snake would be better